### ISSUE TITLE
Reduce size/dependencies?

### ISSUE THREAD

**Issue by jimmywarting** [feross/typedarray-to-buffer#22]
You could remove `is-typedarray`?
and use offset+length from the start instead of slicing afterwards

```js
module.exports = function typedarrayToBuffer (arr) {
  return ArrayBuffer.isView(arr)
    ? Buffer.from(arr.buffer, arr.byteOffset, arr.byteOffset + arr.byteLength)
    : Buffer.from(arr)
}
```

**Comment by feross** [comment]
I think it should actually be:

```js
module.exports = function typedarrayToBuffer (arr) {
  return ArrayBuffer.isView(arr)
    ? Buffer.from(arr.buffer, arr.byteOffset, arr.byteLength)
    : Buffer.from(arr)
}
```

But you're right. The browser support has improved enough that we can probably simplify this package a lot!

**Comment by feross** [comment]
Released as 4.0.0
